packages/miniflare/turbo.json

```diff
@@ -1,15 +1,16 @@
 {
 	"$schema": "http://turbo.build/schema.json",
 	"extends": ["//"],
-	"pipeline": {
+	"tasks": {
 		"build": {
 			"inputs": ["src/**", "scripts/**", "types/**", "*.mjs", "*.js", "*.json"],
 			"outputs": ["dist/**", "bootstrap.js", "worker-metafiles/**"],
-			"env": ["CI_OS"]
+			"env": ["CI_OS", "NODE_EXTRA_CA_CERTS"]
```

Comment:
without this, the miniflare test `Miniflare: HTTPS fetches using browser CA certificates` will fail